### PR TITLE
Fix for disabling triton templates

### PR DESCRIPTION
**Pull request by jansel**
Some tests started failing after
https://github.com/pytorch/torchdynamo/pull/422

### CODE REVIEW

File: torchinductor/scheduler.py [hunk 1]
```diff
@@ -142,7 +147,7 @@ def run(self, codegen_extern_call):
         self.allocate()
         self.scheduler.run_count += 1
         self.scheduler.pending_buffer_names.add(self.get_name())
-        if type(self.node) not in TemplateKernels:
+        if should_use_template(self.node):
```
Comment (pyjhzwh) [inline torchinductor/scheduler.py:150]: `if not should_use_template(self.node)`

File: torchinductor/scheduler.py [hunk 2]
```diff
@@ -19,15 +19,20 @@
 from .sizevars import SimplifyIndexing
 from .virtualized import V
 
-TemplateKernels = [ir.Convolution]
+template_kernels = [ir.Convolution]
```
Comment (pyjhzwh) [inline torchinductor/scheduler.py:22]: I think we could set template_kernels = [] to disable TritonTemplateKernel

Comment (jansel) [reply torchinductor/scheduler.py:22]: This is defined before the configuration might get changed, so we will need to move it out of global scope for that to work.

Comment (pyjhzwh) [reply torchinductor/scheduler.py:22]: oh I missed the config. Makes sense

**Pull request merged**